adafruit_atecc/adafruit_atecc.py

```diff
@@ -248,7 +260,7 @@ def lock_all_zones(self):
         self.lock(0)
         self.lock(1)
 
-    def lock(self, zone):
+    def lock(self, zone: Any):
```

Comment:
`zone` is typed as `int` in the docstring just below here. Are there cases where it would be some type other than `int`? We change `Any` to `int` if not